pkg/responses/account.go

```diff
@@ -0,0 +1,9 @@
+package responses
+
+import "github.com/gofiber/fiber"
+
+type Account struct {
```

Comment:
I like that you have decoupled the response from the actual model. But it should not belong to the pkg.
This is tied to the fiber, and it should be a child package of the handler that uses this one.
pkg directory is always debatable, some people like to use it, some people don't. But what is common about it is that it should provide library-like way of dealing with things. Meaning that it should be reusable from any part of your project. Here, this is tied to the fiber so it is only usable by handlers.